docs/weighting/travelSurveyWeighting.md

```diff
@@ -0,0 +1,42 @@
+# Weighting in Travel Surveys: Making Samples Representative
+
+**Rationale** Travel survey weighting is not processed by Transition, but a short introduction is useful to understand how weighting work and why having weighted input data is important when available.
```

Comment:
"Travel survey weighting is not processed by Transition"  que veux-tu dire par ça ??

Reply:
Ça veut juste dire que les données OD que tu importes doivent être déjà pondérées. Transition pourra lire le poids en choisissant la bonne colonne lors de l'import.